src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getDateFormatted, getPercentFormatted } from '../utils/helpers';
+
+class UserInfoSection extends React.Component {
+  render() {
+    const { userData } = this.props;
+
+    // Used to get today's date to show information current of
+    const today = new Date();
+    const todayFormatted = getDateFormatted(today);
+
+    // Check if percent is a number
+    let percentageBenefit = 'unavailable';
+    if (getPercentFormatted(userData.percentageBenefit)) {
+      percentageBenefit = getPercentFormatted(userData.percentageBenefit);
+    }
+
+    return (
+      <div className="gibstatus">
+        <h3 className="section-header">{userData.firstName} {userData.lastName}</h3>
```

Comment:
Also just noticed that the UI for pages (1) and (2) when there is no enrollment history available includes the header defined here showing the user's name, but for (3) when enrollment history is available, the user's name isn't in a header element; just listed with the data:
![image](https://user-images.githubusercontent.com/25439097/26905538-c7637070-4b9c-11e7-9bcc-b4e828115de0.png)

Going off this document:
https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/EVSS%20Integration/Post-911%20Enrollment%20Status/design/post-911_complete_UI.pdf